package.json

```diff
@@ -1,32 +1,16 @@
 {
-  "name": "@node-saml/node-saml",
+  "name": "@foxden/node-saml",
```

Comment:
Other than the `name` and `repository` fields, we should generally leave this file alone - especially, the `license` section. MIT license is pretty open.